nx.json

```diff
@@ -10,7 +10,7 @@
     "default": {
       "runner": "@nrwl/nx-cloud",
       "options": {
-        "cacheableOperations": ["build", "test", "lint", "package", "build-optimized"],
```

Comment:
unsure what this is about... :(

Reply:
For some reason NX kept skipping to build packages, thinking the output is the same:
https://app.circleci.com/pipelines/github/storybookjs/storybook/24034/workflows/7948e6a5-0fe5-474e-87d1-4b5e52cc9e61/jobs/366738